.gitlab-ci.yml

```diff
@@ -86,10 +84,24 @@ default:
     - rustup +nightly show
     - cargo +nightly --version
 
+.rusty-cachier:
+  before_script:
+    - curl -s https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.parity.io/parity/infrastructure/ci_cd/rusty-cachier/client/-/raw/release/util/install.sh | bash
```

Comment:
we shall put this into a CI image afterwards

Reply:
We explicitly grab latest `rusty-cachier` release each time on purpose. It allows us to roll out fixes for it instantly. IMO better leave it as is.